include/ck_tile/core/arch/mma/wmma/wmma.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright © Advanced Micro Devices, Inc., or its affiliates.
+// SPDX-License-Identifier:  MIT
+
+#pragma once
+
+namespace ck_tile::core::arch::mma {
+
+/*! @enum WmmaCtrlFlags
+ *  @brief Common wmma control flags for gfx11 and gfx12
+ */
+enum struct WmmaCtrlFlags : bool
```

Comment:
Is this pattern going to cause trouble for utilities like iwyu? It's probably better not to put code ahead of #include files.